code/modules/cargo/packs/organic.dm

```diff
@@ -299,7 +264,12 @@
 					/obj/item/reagent_containers/food/snacks/grown/mushroom/chanterelle,
 					/obj/item/reagent_containers/food/snacks/grown/onion,
 					/obj/item/reagent_containers/food/snacks/grown/pumpkin)
-	crate_name = "food crate"
+	crate_name = "veggie crate"
+
+/datum/supply_pack/organic/randomized/fill(obj/structure/closet/crate/C)
+	for(var/i in 1 to 15)
```

Comment:
This should become:
```
	for(var/i in 1 to num_contained)
```
After following my suggestion on the randomised candies pack.

Also please move this proc next to the `/organic` parent type, it's shared by many of these supply packs and it's pretty hard to find right now.